server/implementation-cdi/src/main/java/io/smallrye/graphql/cdi/config/GraphQLConfig.java

```diff
@@ -46,6 +46,10 @@
     @ConfigProperty(name = ConfigKey.ENABLE_TRACING, defaultValue = "false")
     private boolean tracingEnabled;
 
+    @Inject
+    @ConfigProperty(name = ConfigKey.ENABLE_VALIDATION, defaultValue = "false")
+    private boolean validationEnabled;
+
```

Comment:
Maybe the default can be true ?

Reply:
Only if a BeanValidation implementation is on the classpath. Maybe we could be smarter about this and find out at runtime. I'm not sure how this would work on Quarkus, though.

I personally would also like to have Metrics and Tracing enabled if the libraries are available, but I understand that some people will argue about a performance impact.

Reply:
O yes true. So let's keep it on false.